subsys/net/lib/aws_fota/src/aws_fota_json.c

```diff
@@ -36,19 +47,12 @@ struct execution_obj {
 };
 
 struct notify_next_obj {
+	const char *client_token;
 	int timestamp;
 	struct execution_obj execution;
 };
 
-struct status_details_obj {
-	const char *next_state;
-};
 
-struct execution_state_obj {
-	const char *status;
-	struct status_details_obj status_details;
-	int version_number;
-};
 
```

Comment:
While you are here, you can fix the excess newlines in this file (here among other places)

Reply:
I don't get this comment there is no newline marked by it only something which was moved

Reply:
Tried to fix some of the new lines so it looks a bit uglier IMO but if you have a small console I guess it will be easier to edit.